JSQMessagesViewController/Layout/JSQMessagesBubblesSizeCalculator.m

```diff
@@ -138,4 +145,28 @@ - (CGSize)jsq_avatarSizeForMessageData:(id<JSQMessageData>)messageData
     return layout.incomingAvatarViewSize;
 }
 
+- (NSInteger)magixInsetAddition {
+    //  Creating a getter for this magix value because we are using it in a couple of places.
+    //
+    //  add extra 2 points of space, because `boundingRectWithSize:` is slightly off
+    //  not sure why. magix. (shrug) if you know, submit a PR
+    return 2;
+}
+
+- (CGFloat)textBubbleWidth:(JSQMessagesCollectionViewFlowLayout *)layout
```

Comment:
![img](http://www.thewalkingdead.com/wp-content/uploads/2013/04/shia-magic.gif)

sometimes you gotta add a little magix.